README.md

```diff
@@ -153,6 +153,9 @@ has been under active development since then.
 For usage questions, the best place to go to is [StackOverflow](https://stackoverflow.com/questions/tagged/pandas).
 Further, general questions and discussions can also take place on the [pydata mailing list](https://groups.google.com/forum/?fromgroups#!forum/pydata).
 
+## Notes
+If you are using Jupyter notebook or Google's Colaboratory then pandas are pre-installed in it.
```

Comment:
Is this true for Jupyter? I believe it's possible to have Jupyter installed without pandas, at the very least via `conda remove --force pandas`.

Reply:
or if you install Jupyter without going through conda. Thanks for the suggestion, but IMO this doesn't need to be in the README (though perhaps a note on google colab would go well somewhere else in the docs)